### ISSUE TITLE
Double Click to Keep Open

### ISSUE THREAD

**Issue by TomasJuocepis** [eamodio/vscode-gitlens#1122]
In the sidebar, in 'REPOSITORIES' section, in the changed files tree, when I select a file, an editor tab opens up with the diff of that file with the title in italics (not sure how it's called, preview mode?). I would like to be able to then double click my mouse on that same selected file to perform the 'Keep Open' (CTRL+K ENTER) action on it. Right now I have to move the cursor to the newly opened tab title and double click there (or use a keyboard shortcut).

**Comment by fsackur** [comment]
Yes, this is "preview mode". FWIW, these are globally configurable in Code with `workbench.editor.enablePreview` and `workbench.editor.enablePreviewFromQuickOpen`.

However, I too would like this feature in GitLens.

**Comment by eamodio** [comment]
This can't easily be implemented in GitLens itself, but we are working on enabling this for all extensions here: https://github.com/microsoft/vscode/issues/85636. Once that lands, then GitLens can take advantage of it.

**Comment by GlebMorgan** [comment]
Seems like https://github.com/microsoft/vscode/issues/85636 has been implemented and tested 3 years ago, so can this feature be implemented now in GitLens?  This will be an extremely important addition.
